package.json

```diff
@@ -15,14 +15,12 @@
     "format:staged": "pretty-quick --staged",
     "predev": "npm run build",
     "dev": "http-server -o example/index.html -s -c-1 -p 8080",
+    "build:ts": "tsup",
```

Comment:
I had to use an external config file for tsup because it was the only way I found to change the name of the outfile (to include `.min`). Personally, I wouldn't keep `.min` in the filename, unless you ship both the unminified and minified versions of the library. But we can keep it if you have strong feelings.

Reply:
I partially agree with this, and this circles back to our previous conversation about including an unaltered source file for code inspection in IDEs. Keep in mind that If you are going to remove the `.min` from the file name, the documentation should be updated to reflect this change. I don't mind an external config for a build tool, and I prefer the `.min` suffix in any code that is minified (due to how it's treated in IDE file trees).

Reply:
Fine by me, I'll leave the `.min` suffix